next/components/forms/info-components/Tooltip/TooltipPopup.tsx

```diff
@@ -51,7 +51,9 @@ const TooltipPopup = ({ arrow = true, className, text, position = 'top-left' }:
           className,
         )}
       >
-        <p className="w-max">{text}</p>
+        <p className="w-max">
+          <ReactMarkdown>{text || ''}</ReactMarkdown>
```

Comment:
Please use "AccountMarkdown" (merged just today), it should ensure that we have correct typography weights (and helps us with fixing them across every markdown usage should they change)